src-frontend/app/components/procedure-card.js

```diff
@@ -1,5 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-    tagName: 'li'
+    tagName: 'li',
+
+    actions: {
+        deleteProcedure: function(procedure) {
+            procedure.deleteRecord();
+            procedure.save();
```

Comment:
We are using this convention elsewhere. Want me to add it to this PR or a separate one?